### PR TITLE
Add /iot/smart-home

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Added /smart-home to /internet-of-things bubble based on [copy doc](https://docs.google.com/document/d/1zwB-6Flt0Jm7XIxhWUWRJjKC1WCGVmVVNmPHtyOTHuE/edit#)
- Added new contact form based on [copy doc](https://docs.google.com/document/d/1a_OLqNDCkxxS7EHPV0Q1clCZru8h5FQUqljVWMgW1Og/edit#)

## QA

- visit https://ubuntu-com-11781.demos.haus/internet-of-things/smart-home
- See that it matches the copy doc
- Click "Get in touch", see that the form submits correctly

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5370

### CODE REVIEW

File: templates/internet-of-things/smart-home.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
+{% extends "internet-of-things/base_things.html" %}
+
+
+{% block title %}Open source smart home | Ubuntu for the Internet of Things {% endblock %}
+{% block meta_description %}The open source smart home is easier, faster, and more secure with Ubuntu. Linux in the smart home is here to stay, and Canonical is here to guide you on your journey. Ubuntu Core is the operating system for the smart home.{% endblock meta_description %}
+
+{% block meta_copydoc %}{% endblock meta_copydoc %}
```
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:7]:
```suggestion
{% block meta_copydoc %}https://docs.google.com/document/d/1zwB-6Flt0Jm7XIxhWUWRJjKC1WCGVmVVNmPHtyOTHuE/edit#{% endblock meta_copydoc %}
```

File: navigation.yaml [hunk 2]
```diff
@@ -1239,6 +1239,8 @@ iot:
       path: /internet-of-things/networking
     - title: Smart city
       path: /internet-of-things/smart-city
+    - title: Smart home
```
Comment (petesfrench) [inline navigation.yaml:1242]: I thought we weren't adding more things to large navigation's like this one?

File: templates/internet-of-things/smart-home.html [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Secure, resilient, low-touch</h2>
+  </div>
+  <div class="row">
+    <div class="col-6">
+      <h3>Data privacy assured</h3>
+      <p>Keep customer data safe with Snaps. Applications in Ubuntu Core are fully confined, so security threats in one application cant affect other parts of the system.</p>
```
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:60]:
```suggestion
      <p>Keep customer data safe with Snaps. Applications in Ubuntu Core are fully confined, so security threats in one application can't affect other parts of the system.</p>
```

File: templates/internet-of-things/smart-home.html [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+            ) | safe
+          }}
+        </div>
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Snap your application or let us do the heavy lifting</h3>
+
+        <div class="p-stepped-list__content">
+          <p>Getting started with Snap is a breeze. Canonical is on this journey with you, and can give you the assurance that your project won't hit roadblocks down the line. When you're ready to put all the pieces together, we're here to help.</p>
```
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:156]:
```suggestion
          <p><a href="https://snapcraft.io/first-snap">Getting started with Snap</a> is a breeze. Canonical is on this journey with you, and can give you the assurance that your project won't hit roadblocks down the line. When you're ready to put all the pieces together, we're here to help.</p>
```

File: templates/internet-of-things/smart-home.html [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+        </div>
+      </li>
+
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">Port to certified hardware or get support for custom boards</h3>
+
+        <div class="p-stepped-list__content">
+          <p>Once your hardware is ready, it's easy to move your application from the development board to your production hardware. There is a library of hardware that comes pre-certified for Ubuntu, and Canonical is here to help with more customised use cases.</p>
```
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:164]:
```suggestion
          <p>Once your hardware is ready, it's easy to move your application from the development board to your production hardware. There is a <a href="/certified">library of hardware</a> that comes pre-certified for Ubuntu, and Canonical is here to help with more customised use cases.</p>
```

**Comment by petesfrench** [comment]
In the copydoc Dave has suggested the addition of a contextual footer on `/smart-home`

File: templates/internet-of-things/smart-home.html [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+          <p>The <a href="https://snapcraft.io/search?category=devices-and-iot">Snap Store</a> includes a huge collection of open source projects that can be seamlessly integrated into your application. Each snap comes with the same level of OTA and revision control you have for your own application.</p>
+
+
+          {{ image (
+            url="https://assets.ubuntu.com/v1/d22c1230-snap-store-iot-devices.png",
+            alt="",
+            width="636",
+            height="346",
+            hi_def=True,
+            loading="lazy"
```
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:146]:
```suggestion
            loading="lazy",
            attrs={"class": "u-hide--small"}
```
It is difficult to see the image on small screens, so we might as well hide it

File: templates/internet-of-things/smart-home.html [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+
+<section class="p-strip--suru-topped">
+  <div class="row">
+    <div class="col-6">
+      <h1>Reduce development cost for the open source smart home with Ubuntu</h1>
+      <p class="p-heading--4">Shorter development cycles, effortless maintenance</p>
+
+      <p>Product development for the smart home comes with unique challenges: margins can be thin, and time-to-market is critical. Canonical can provide the support and security maintenance you need so your team can focus on developing world-class products.</p>
+      <p>
+        <a href="#get-in-touch" class="p-button--positive">Contact us to get started</a>
```
Comment (petesfrench) [inline templates/internet-of-things/smart-home.html:19]:
```suggestion
        <a href="#get-in-touch" class="js-invoke-modal p-button--positive">Contact us to get started</a>
```

File: templates/shared/forms/interactive/smart-home.html [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+                <input required id="firstName" name="firstName" maxlength="255" type="text" required="required" />
+              </li>
+              <li class="p-list__item">
+                <label for="lastName">Last name:</label>
+                <input required id="lastName" name="lastName" maxlength="255" type="text" required="required" />
+              </li>
+              <li class="p-list__item">
+                <label for="company">Company:</label>
+                <input required id="company" name="company" maxlength="255" type="text" required="required" />
+              </li>
```
Comment (petesfrench) [inline templates/shared/forms/interactive/smart-home.html:91]: ```suggestion```

Comment (petesfrench) [inline templates/shared/forms/interactive/smart-home.html:91]:
```suggestion

```

File: templates/shared/forms/interactive/smart-home.html [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+              <li class="p-list__item">
+                <label for="email">Work email:</label>
+                <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" required="required" />
+              </li>
+              <li class="p-list__item">
+                <label for="phone">Mobile/cell phone number:</label>
+                <input required id="phone" name="phone" maxlength="255" type="tel" required="required" />
```
Comment (petesfrench) [inline templates/shared/forms/interactive/smart-home.html:98]:
```suggestion
                <input id="phone" name="phone" maxlength="255" type="tel" required="required" />
```

File: navigation.yaml (hunk 2 above)
Comment (sowasred2012) [reply navigation.yaml:1242]: @davegoddard42 and I spoke about it yesterday, and he is fine with this temporarily - he's going to be looking at a new design for the new link sections on /iot and the subpages to address this.

**Comment by davegoddard42** [comment]
LGTM @sowasred2012

**Pull request merged**